.vib/kong/vib-verify.json

```diff
@@ -22,7 +22,7 @@
+        "runtime_parameters": "cmVwbGljYUNvdW50OiAxCmRhdGFiYXNlOiBwb3N0Z3Jlc3FsCmNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICBlbmFibGVkOiB0cnVlCiAgcnVuQXNVc2VyOiAxMDAyCmtvbmc6CiAgY29udGFpbmVyUG9ydHM6IAogICAgcHJveHlIdHRwOiA4MDAwCiAgICBwcm94eUh0dHBzOiA4NDQzCiAgICBhZG1pbkh0dHA6IDgwMDEKICAgIGFkbWluSHR0cHM6IDg0NDQKc2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBleHBvc2VBZG1pbjogdHJ1ZQogIHBvcnRzOgogICAgcHJveHlIdHRwOiA4MAogICAgcHJveHlIdHRwczogODQ0MwogICAgYWRtaW5IdHRwOiA0NDMKICAgIGFkbWluSHR0cHM6IDg0NDQKaW5ncmVzc0NvbnRyb2xsZXI6CiAgZW5hYmxlZDogdHJ1ZQogIGluZ3Jlc3NDbGFzczoga29uZwpwb3N0Z3Jlc3FsOgogIGVuYWJsZWQ6IHRydWUKICBhdXRoOgogICAgdXNlcm5hbWU6IGtvbmcKICAgIHBhc3N3b3JkOiAiIgogICAgZGF0YWJhc2U6IGtvbmcKZXh0cmFEZXBsb3k6Ci0gYXBpVmVyc2lvbjogYXBwcy92MQogIGtpbmQ6IERlcGxveW1lbnQKICBtZXRhZGF0YToKICAgIG5hbWU6IG5naW54LXZpYi10ZXN0cwogIHNwZWM6CiAgICByZXBsaWNhczogMQogICAgc2VsZWN0b3I6CiAgICAgIG1hdGNoTGFiZWxzOgogICAgICAgIGFwcDogbmdpbngKICAgIHRlbXBsYXRlOgogICAgICBtZXRhZGF0YToKICAgICAgICBsYWJlbHM6CiAgICAgICAgICBhcHA6IG5naW54CiAgICAgIHNwZWM6CiAgICAgICAgY29udGFpbmVyczoKICAgICAgICAtIG5hbWU6IG5naW54CiAgICAgICAgICBpbWFnZTogYml0bmFtaS9uZ2lueAogICAgICAgICAgcG9ydHM6CiAgICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDgwODAKLSBhcGlWZXJzaW9uOiB2MQogIGtpbmQ6IFNlcnZpY2UKICBtZXRhZGF0YToKICAgIG5hbWU6IG5naW54LXZpYi10ZXN0cwogICAgbGFiZWxzOgogICAgICBhcHA6IG5naW54CiAgc3BlYzoKICAgIHNlbGVjdG9yOgogICAgICBhcHA6IG5naW54CiAgICBwb3J0czoKICAgIC0gcG9ydDogODA4MAogICAgICBuYW1lOiBodHRwCiAgICAgIHRhcmdldFBvcnQ6IDgwODAKLSBhcGlWZXJzaW9uOiB2MQogIGtpbmQ6IFNlY3JldAogIG1ldGFkYXRhOgogICAgbmFtZTogYXBpa2V5LXZpYi10ZXN0cwogIGRhdGE6CiAgICBrb25nQ3JlZFR5cGU6IGEyVjVMV0YxZEdnPQogICAga2V5OiBRMjl0Y0d4cFkyRjBaV1JRWVhOemQyOXlaREV5TXlFMAotIGFwaVZlcnNpb246IGNvbmZpZ3VyYXRpb24ua29uZ2hxLmNvbS92MQogIGtpbmQ6IEtvbmdDb25zdW1lcgogIG1ldGFkYXRhOgogICAgbmFtZTogdmliLXRlc3RzCiAgICBhbm5vdGF0aW9uczoKICAgICAga3ViZXJuZXRlcy5pby9pbmdyZXNzLmNsYXNzOiBrb25nCiAgdXNlcm5hbWU6ICJ2aWItdGVzdHMiCiAgY3JlZGVudGlhbHM6CiAgLSBhcGlrZXktdmliLXRlc3RzCi0gYXBpVmVyc2lvbjogY29uZmlndXJhdGlvbi5rb25naHEuY29tL3YxCiAga2luZDogS29uZ1BsdWdpbgogIG1ldGFkYXRhOgogICAgbmFtZTogYXV0aC12aWItdGVzdHMKICBwbHVnaW46IGtleS1hdXRoCi0gYXBpVmVyc2lvbjogbmV0d29ya2luZy5rOHMuaW8vdjEKICBraW5kOiBJbmdyZXNzCiAgbWV0YWRhdGE6CiAgICBuYW1lOiB2aWItdGVzdHMKICAgIGFubm90YXRpb25zOgogICAgICBrb25naHEuY29tL3N0cmlwLXBhdGg6ICJ0cnVlIgogICAgICBrb25naHEuY29tL3BsdWdpbnM6IGF1dGgtdmliLXRlc3RzCiAgc3BlYzoKICAgIGluZ3Jlc3NDbGFzc05hbWU6IGtvbmcKICAgIHJ1bGVzOgogICAgLSBob3N0OiB3d3cuZXhhbXBsZS5jb20KICAgICAgaHR0cDoKICAgICAgICBwYXRoczoKICAgICAgICAtIHBhdGg6IC9uZ2lueAogICAgICAgICAgcGF0aFR5cGU6IEltcGxlbWVudGF0aW9uU3BlY2lmaWMKICAgICAgICAgIGJhY2tlbmQ6CiAgICAgICAgICAgIHNlcnZpY2U6CiAgICAgICAgICAgICAgbmFtZTogbmdpbngtdmliLXRlc3RzCiAgICAgICAgICAgICAgcG9ydDoKICAgICAgICAgICAgICAgIG51bWJlcjogODA4MA==",
```

Comment:
As you are using the http service ports in Cypress and Goss, I would add `service.disableHttpPort=false` to the runtime params.

One doubt regarding the DB credentials, are we not able to change the default values?